meetings/agenda.md

```diff
@@ -7,3 +7,5 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to Discuss
 
+- Adoping a code coverage tool: I'm a big fan of using CodeCov in my work/personal projects mainly because it fails CI if project or patch coverage declines. I accept the coverage is not the best quality metric in the world, but the PR comments from CodeCov _(Which I'm most familiar with)_ provide pretty swift feedback on submitted code… I'm fairly confident that you can install the CodeCov app org wide and a token is not required for upload with public repos, theoretically meaning that the martix action could be updated to collect and upload coverage for all repos without any adjustment on a per-repository basis.
```

Comment:
Rather than CodeCov, I'd really like to see mutation testing put in place, especially on added/removed code.

Coverage itself is a very weak metric, IMO. 

I'm also fine with gradually incrementing our coverage reports: what matters most to me is that each change is incrementally verified, and that PR authors get automatic feedback on uncovered code they add/change, so we don't need to chase them asking for tests.